racingcar-java/src/test/java/mission02/CalculationTest.java

```diff
@@ -0,0 +1,78 @@
+package mission02;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.Arrays;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class CalculationTest {
+
+    Calculation calculation = new Calculation();
+    String[] types = {OperatorType.PLUS, OperatorType.MINUS, OperatorType.MULTIPLY, OperatorType.DIVIDED};
+
+    @DisplayName("연산기호 테스트 - +, -, *, /")
+    @ParameterizedTest
+    @ValueSource(strings = {"+", "-", "*", "/"})
+    void 연산기호가_올바른_사칙연산의_연산기호인지_확인한다(String type){ // 이거 리뷰 필요
+        // when
+        boolean check = Arrays.stream(types).anyMatch(type::equals);
```

Comment:
```suggestion
        boolean check = Arrays.stream(types)
            .anyMatch(type::equals);
```
`.` 구조는 뎁스를 내려주는게 컨벤션이에요